Source/WebKit/GPUProcess/media/RemoteAudioSessionProxyManager.cpp

```diff
@@ -213,6 +213,22 @@ void RemoteAudioSessionProxyManager::updatePresentingProcesses()
     AudioSession::sharedSession().setPresentingProcesses(WTFMove(presentingProcesses));
 }
 
+void RemoteAudioSessionProxyManager::beginInterruptionRemote()
+{
+    auto& session = this->session();
+    session.removeInterruptionObserver(*this);
+    session.beginInterruption();
+    session.addInterruptionObserver(*this);
```

Comment:
that feels a bit messy and difficult to understand at first glance, it deserves some comment

Reply:
This is also explained in the ChangeLog 😅

Reply:
But I am happy to leave a comment here too

Reply:
Added comment.